### DISCUSSION TITLE
Project IDX setup to start the Get Started

### DISCUSSION THREAD

**Discussion by giacomofm** [vercel/next.js#67960] [Show and tell]
Hi everyone,

I'm a seasoned backend developer, and I've recently started learning Next.js by following the dashboard-app tutorial (really well-crafted). To streamline the setup process for those who don't have a development environment ready on their PC or can't have one, I've created a mini project utilizing Google's Project IDX.

You can find it here: https://github.com/giacomofm/idx-nextjs-learn-dashboard-app

This project is designed to help people set up everything needed and follow the Next.js tutorial seamlessly. If you're in the same boat as me, just starting out with Next.js, or if you simply want an easier way to get started, check it out!

If you try it out, let me know your thoughts or if you encounter any strange behavior.

Happy coding!

**Comment by randomhumandownhere** [comment]
for anyone new coming here, project idx is now firebase studio in case you're confused and it still works like a charm
